tools/common/Frameworks.cs

```diff
@@ -422,7 +423,8 @@ public static Frameworks CreateiOSFrameworks (bool is_simulator_build)
 
 				{ "CoreLocationUI", "CoreLocationUI", 15,0 },
 
-				{ "Chip", "CHIP", new Version (15, 0), NotAvailableInSimulator /* no headers in beta 2 */ }
+				{ "Chip", "CHIP", new Version (15, 0), NotAvailableInSimulator /* no headers in beta 2 */ },
+				{ "Phase", "PHASE", new Version (15,0), NotAvailableInSimulator /* no headers in beta 2 */ },
```

Comment:
is not a separate entry required for MacCatalyst ?

Reply:
Mac Catalyst will pick up the iOS frameworks automatically.

Reply:
https://github.com/xamarin/xamarin-macios/blob/524fd24022e8e6e0f95caa22f10029a97b3b0e95/tools/common/Frameworks.cs#L594-L597